src/cdk/explorer/nodes/constructNode.ts

```diff
@@ -24,7 +24,7 @@ export class ConstructNode extends AWSTreeNodeBase {
         public readonly construct: ConstructTreeEntity
     ) {
         super(construct.id, collapsibleState)
-        this.contextValue = 'awsCdkConstructNode'
```

Comment:
You're conditionally running one of two expressions here. Since in both cases we are assigning the value for `this.contextValue`, we can update this to have the condition determine which of the two values gets assigned.

`this.contextValue = treeInspector.isStateMachine(construct) ? 'awsCdkStateMachineNode' : 'awsCdkConstructNode'`